src/ArduinoCellular.cpp

```diff
@@ -1,6 +1,9 @@
 
 
 #include "ArduinoCellular.h"
+#if defined(ARDUINO_ARCH_MBED)
+  #include "Watchdog.h"
+#endif
```

Comment:
Does this flag also cover https://github.com/arduino/ArduinoCore-renesas and not just https://github.com/arduino/ArduinoCore-mbed?

Reply:
No it is mbed only. This change is related to ArduinoIotCloud integration that enables watchdog for mbed and samd boards but not for the renesas one.